lib/_http_server.js

```diff
@@ -452,6 +454,11 @@ function storeHTTPOptions(options) {
     validateInteger(maxHeaderSize, 'maxHeaderSize', 0);
   this.maxHeaderSize = maxHeaderSize;
 
+  const fastDump = options.fastDump;
+  if (fastDump !== undefined)
+    validateBoolean(fastDump, 'options.fastDump');
+  this[kfastDump] = fastDump || false;
```

Comment:
The OR seems redudant since we are passing through `validateBoolean`

Reply:
How so? if fastDump is undefined, `this[kfastDump]` will be undefined instead of false.